app/assets/stylesheets/darkswarm/footer.scss

```diff
@@ -4,8 +4,8 @@
 
 footer {
   .row {
-    p a {  /* was p a  */
-      font-size: 1rem; /* was 0.875rem */
+    p a {
```

Comment:
I think this commit should be squashed with the previous one (to have a clean history). See https://github.com/openfoodfoundation/openfoodnetwork/wiki/Making-a-great-pull-request#keep-your-commit-history-clean-tidy-and-up-to-date for more.